docs/references/vod/upload.mdx

```diff
@@ -43,14 +73,34 @@ curl --location --request POST 'https://livepeer.studio/api/asset/request-upload
 
 ```jsx
 {
-    "url": "https://origin.livepeer.studio/api/asset/upload/aHR0cHM6Ly9zdG9yYWdlLmdvb2dsZWFwaXMuY29tL2xwLW55Yy12b2QtbW9uc3Rlci9kaXJlY3RVcGxvYWQvNjI3OHQyZWlrY2JqZXUxNC9zb3VyY2U%2F...",
+    "url": "https://origin.livepeer.com/api/asset/upload/direct?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwbGF5YmFja0lkIjoiZWViOWlsajBmODRnMzRueSIsInByZXNpZ25lZFVybCI6Imh0dHBzOi8vc3RvcmFnZS5nb29nbGVhcGlzLmNvbS9scC11cy12b2QtY29tL2RpcmVjdFVwbG9hZC9lZWI5aWxqMGY4NGczNG55P1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNvbnRlbnQtU2hhMjU2PVVOU0lHTkVELVBBWUxPQUQmWC1BbXotQ3JlZGVudGlhbD1HT09HMUVZVU5XTlY2UllJSzU0MkhXQTNSTDdSQjdKVU9FUTNZTDIzUVlCNkNIUEc1SE8yUUMzM01KVlVZJTJGMjAyMjA4MDUlMkZ1cyUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMjA4MDVUMTYwMjUyWiZYLUFtei1FeHBpcmVzPTkwMCZYLUFtei1TaWduYXR1cmU9OTNjYmQ2MTM1ZDE0OGY5NWExMjdiNDdmMDM5Njk4NTQ1ODg4ZTM3YzkyMzZjMTQ3Y2MyYzhjOTdhYTViYjNmNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmeC1pZD1QdXRPYmplY3QiLCJhdWQiOiJodHRwczovL2xpdmVwZWVyLmNvbSIsImlhdCI6MTY1OTcxNTM3Mn0.ifvnEp50JnlOqP0HAD9duuFkLG3QGtuwRI4mxOQFLKs",
+    "tusEndpoint": "https://origin.livepeer.com/api/asset/upload/tus?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwbGF5YmFja0lkIjoiZWViOWlsajBmODRnMzRueSIsInByZXNpZ25lZFVybCI6Imh0dHBzOi8vc3RvcmFnZS5nb29nbGVhcGlzLmNvbS9scC11cy12b2QtY29tL2RpcmVjdFVwbG9hZC9lZWI5aWxqMGY4NGczNG55P1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNvbnRlbnQtU2hhMjU2PVVOU0lHTkVELVBBWUxPQUQmWC1BbXotQ3JlZGVudGlhbD1HT09HMUVZVU5XTlY2UllJSzU0MkhXQTNSTDdSQjdKVU9FUTNZTDIzUVlCNkNIUEc1SE8yUUMzM01KVlVZJTJGMjAyMjA4MDUlMkZ1cyUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMjA4MDVUMTYwMjUyWiZYLUFtei1FeHBpcmVzPTkwMCZYLUFtei1TaWduYXR1cmU9OTNjYmQ2MTM1ZDE0OGY5NWExMjdiNDdmMDM5Njk4NTQ1ODg4ZTM3YzkyMzZjMTQ3Y2MyYzhjOTdhYTViYjNmNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmeC1pZD1QdXRPYmplY3QiLCJhdWQiOiJodHRwczovL2xpdmVwZWVyLmNvbSIsImlhdCI6MTY1OTcxNTM3Mn0.ifvnEp50JnlOqP0HAD9duuFkLG3QGtuwRI4mxOQFLKs",
```

Comment:
Let's omit these tokens, not very helpful to have these huge strings in the docs anyway. 

You can use this for a fake token in the docs 🌚 
```suggestion
    "url": "https://origin.livepeer.com/api/asset/upload/direct?token=eyJhbGciOiJIUzI1NiJ9.eyJtc2ciOiJoZWxsbyBoYWNrZXIsIHRoZXJlJ3Mgbm90aGluZyBmb3IgeW91IGhlcmUg8J-YhiJ9.1YDjmXsqLcgNyMSzT4kXl_kIni46_EuGX_xfqmC7e0Q",
    "tusEndpoint": "https://origin.livepeer.com/api/asset/upload/tus?token=eyJhbGciOiJIUzI1NiJ9.eyJtc2ciOiJoZWxsbyBoYWNrZXIsIHRoZXJlJ3Mgbm90aGluZyBmb3IgeW91IGhlcmUg8J-YhiJ9.1YDjmXsqLcgNyMSzT4kXl_kIni46_EuGX_xfqmC7e0Q"
```